### ISSUE TITLE
Safari browser, does not cover the entire text!

### ISSUE THREAD

**Issue by Bahaa-Fadhil** [openlayers/openlayers#14621]
**Describe the bug**
In our application, we use openLayers to display site names on a map. The text/name is displayed as a label on the map using the openLayers Text style. We have tested the solution on different browsers such as Chrome, Edge, and Safari. On Chrome and Edge, the solution works correctly, but on Safari, we have observed a bug.

The bug is that the name/ text label, which uses the openLayers Text style, does not cover the entire text on Safari browser, while on Chrome and Edge, the text covers the entire label. This means that the site name is not fully displayed on Safari, which could potentially cause confusion for users trying to identify a site on the map.

To summarize, the issue is that the openLayers Text style is not working correctly on Safari, causing the site label to be displayed incompletely.

**Illustration pictures of bugs**
![image](https://user-images.githubusercontent.com/32929383/228495186-98717b65-4f30-4033-ae71-aa0337365e97.png) ![image](https://user-images.githubusercontent.com/32929383/228494837-eb4422fb-2d0c-467a-84a3-1c14683a0398.png)

**Comment by Bahaa-Fadhil** [comment]
Ok, so the solution is that specifying font sizes in pixels and not on rem or em.

@M393 Thanks for feedback :)
